tests/jerry/es2015/new-target-for-dataview.js

```diff
@@ -0,0 +1,33 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+//
+var buffer = new ArrayBuffer (16);
+try {
+ Reflect.construct (DataView, buffer);
+ assert (false);
+} catch (e) {
+ assert (e instanceof TypeError);
+}
+
+try {
+ Reflect.construct (DataView);
```

Comment:
I think this tc is not related to this feature. I'd add a tc when the 3rd argument is present.